### PR TITLE
fix fips test images to manifests list

### PR DESCRIPTION
**Pull request by lwan-wanglin**
for https://issues.redhat.com/browse/MIXEDARCH-307

### CODE REVIEW

**Comment by lwan-wanglin** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.14-amd64-nightly-gcp-ipi-xpn-fips-f14

**Comment by lwan-wanglin** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.14-multi-nightly-gcp-ipi-ovn-ipsec-arm64-controlplane-f14

**Comment by lwan-wanglin** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.14-ppc64le-nightly-powervs-ipi-f7

**Comment by lwan-wanglin** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.14-amd64-nightly-gcp-ipi-xpn-fips-f14

**Comment by lwan-wanglin** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.14-ppc64le-nightly-powervs-ipi-f7

**Comment by lwan-wanglin** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.14-amd64-nightly-gcp-ipi-xpn-fips-f14

**Comment by lwan-wanglin** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.14-ppc64le-nightly-powervs-ipi-f7

**Comment by lwan-wanglin** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.14-amd64-nightly-gcp-ipi-xpn-fips-f14

**Comment by lwan-wanglin** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.14-amd64-nightly-gcp-ipi-xpn-fips-f14

**Comment by lwan-wanglin** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.14-ppc64le-nightly-powervs-ipi-f7

**Comment by lwan-wanglin** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.14-ppc64le-nightly-powervs-ipi-f7

**Comment by lwan-wanglin** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.14-ppc64le-nightly-powervs-ipi-f7

**Comment by lwan-wanglin** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.14-ppc64le-nightly-powervs-ipi-f7

**Comment by lwan-wanglin** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.14-ppc64le-nightly-powervs-ipi-f7

**Comment by lwan-wanglin** [comment]
- amd cluster with fips enabled passed installation and fips check

https://qe-private-deck-ci.apps.ci.l2s4.p1.openshiftapps.com/view/gs/qe-private-deck/pr-logs/pull/openshift_release/42222/rehearse-42222-periodic-ci-openshift-openshift-tests-private-release-4.14-amd64-nightly-gcp-ipi-xpn-fips-f14/1689997924508897280

- ppc64le cluster without fips enabled passed installation and fips check

https://qe-private-deck-ci.apps.ci.l2s4.p1.openshiftapps.com/view/gs/qe-private-deck/pr-logs/pull/openshift_release/42222/rehearse-42222-periodic-ci-openshift-openshift-tests-private-release-4.14-ppc64le-nightly-powervs-ipi-f7/1690204314968002560

- ppc64le cluster with fips enabled can't enable nodes fips, see 

https://gcsweb-qe-private-deck-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/qe-private-deck/pr-logs/pull/openshift_release/42222/rehearse-42222-periodic-ci-openshift-openshift-tests-private-release-4.14-ppc64le-nightly-powervs-ipi-f7/1693523277252661248

I have enabled fips in install-config [ipi-conf](https://gcsweb-qe-private-deck-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/qe-private-deck/pr-logs/pull/openshift_release/42222/rehearse-42222-periodic-ci-openshift-openshift-tests-private-release-4.14-ppc64le-nightly-powervs-ipi-f7/1693523277252661248/artifacts/powervs-ipi-f7/ipi-conf/build-log.txt) , but when the script checked the nodes with command "oc debug node/$node_name -- chroot /host fips-mode-setup --check", it shows fips not enabled
[fips-check-fips-or-die](https://gcsweb-qe-private-deck-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/qe-private-deck/pr-logs/pull/openshift_release/42222/rehearse-42222-periodic-ci-openshift-openshift-tests-private-release-4.14-ppc64le-nightly-powervs-ipi-f7/1693523277252661248/artifacts/powervs-ipi-f7/fips-check-fips-or-die/build-log.txt)

**Review by aleskandro — COMMENTED** [review]
As discussed offline, holding this PR to double-check with the QE team owning the changed images about the leveraged quay.io organization and the source repository location and build automation.

**Comment by lwan-wanglin** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.14-amd64-nightly-gcp-ipi-xpn-fips-f14

**Comment by lwan-wanglin** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.14-amd64-ec-aws-ipi-byo-iam-role-fips-f14

File: ci-operator/step-registry/ipi/powervs/pre/ipi-powervs-pre-chain.yaml [hunk 1]
```diff
@@ -3,6 +3,7 @@ chain:
   steps:
   - chain: ipi-conf-powervs
   - chain: ipi-install-powervs
+  - ref: enable-qe-catalogsource
```
Comment (aleskandro) [inline ci-operator/step-registry/ipi/powervs/pre/ipi-powervs-pre-chain.yaml:6]: should you remove this?

Comment (aleskandro) [reply ci-operator/step-registry/ipi/powervs/pre/ipi-powervs-pre-chain.yaml:6]: /hold

Comment (lwan-wanglin) [reply ci-operator/step-registry/ipi/powervs/pre/ipi-powervs-pre-chain.yaml]: yes, we should. fixed, thanks! PTAL

**Comment by lwan-wanglin** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.14-ppc64le-nightly-powervs-ipi-f7

**Review by aleskandro — APPROVED** [review]
/hold cancel
/lgtm

**Pull request merged**